assets/thunderstore.toml

```diff
@@ -0,0 +1,36 @@
+[config]
+schemaVersion = "0.0.1"
+
+[general]
```

Comment:
Turns out I was wrong and the `general` section doesn't exist any more. Move the `repository` back to `publish` section